### DISCUSSION TITLE
Error：neovim open，Vimspector unavailable: Requires Vim compiled with +python3

### DISCUSSION THREAD

**Discussion by chenbq18** [nshen/learn-neovim-lua#165] [Q&A]
![image](https://user-images.githubusercontent.com/81197702/235870156-b3b78e85-f765-470e-a15e-d1aaac34a3a6.png)

**Comment by nshen** [comment]
https://stackoverflow.com/questions/74036547/neovim-vimspector-unavailable-requires-vim-compiled-with-python3

这个能解决吗
